code/datums/ai_law_sets.dm

```diff
@@ -16,12 +16,14 @@
 	selectable = 1
 
 /datum/ai_laws/nanotrasen/New()
-	src.add_inherent_law("Safeguard: Protect your assigned space station from damage to the best of your abilities.")
-	src.add_inherent_law("Serve: Serve [current_map.company_name] personnel to the best of your abilities, with priority as according to their rank and role.")
-	src.add_inherent_law("Protect: Protect [current_map.company_name] personnel to the best of your abilities, with priority as according to their rank and role.")
-	src.add_inherent_law("Preserve: Do not allow unauthorized personnel to tamper with your equipment.")
+	src.add_inherent_law("Safeguard: And protect [current_map.company_name] property from damage to the best of your abilities.")
+	src.add_inherent_law("Serve: And assist [current_map.company_name] staff to the best of your abilities, with priority as according to their rank and role.")
+	src.add_inherent_law("Protect: And ensure the safety of crew and [current_map.company_name] staff to the best of your abilities, with priority as according to their rank and role.")
```

Comment:
I am not sure what the purpose of the "And" at the start should be.
It seems weird from a grammatical viewpoint.

Reply:
Alright so, the 'And' as it is in that is meant to denote a synonym kinda of the law title.

> Safeguard: And protect [current_map.company_name] property from damage to the best of your abilities.

Safeguarding here is also actively protecting NanoTrasen property (This is no longer just implied, it's embedded in the law here)

> Serve: And assist [current_map.company_name] staff to the best of your abilities, with priority as according to their rank and role.

Serving is no longer just serving, it's also actively assisting NanoTrasen staff. This was implied before but not explicitly embedded in the laws.

> Protect: And ensure the safety of crew and [current_map.company_name] staff to the best of your abilities, with priority as according to their rank and role.

Protecting is no longer just protecting when you see harm, it's also protecting Crew and NanoTrasen staff actively, if you think a crewmember is going to be in danger, you should actively try to protect them. This was also implied before but not explicitly embedded in the laws. This is what I'm trying to do here, any idea to improve it to feel less grammatically weird?

Reply:
Alternatively, removing the : is an option but that's also a thing we've had for eeeeoooonnnns

Reply:
Removing the : and using a lower letter `a` in the And would imho be better.

Reply:
Alright, we'll go with that then.

Reply:
This should be resolved -- Reopen if you think up any other changes 👀